src/components/patch-assignment-form/components/rows.tsx

```diff
@@ -51,20 +51,54 @@ export const TableRow = ({
   );
   const [validationError, setValidationError] = useState<string | null>(null);
 
-  function onConfirmButtonClick() {
+  const handleConfirmClick = useCallback(() => {
```

Comment:
Im curious, is defining this function expensive enought to cache it?

Reply:
Based on my reasearch yes you can :)